flintrock/flintrock.py

```diff
@@ -179,6 +179,10 @@ def cli(cli_context, config, provider):
 @click.option('--num-slaves', type=int, required=True)
 @click.option('--install-hdfs/--no-install-hdfs', default=False)
 @click.option('--hdfs-version')
+@click.option('--hdfs-download-source',
+              help="Custom URL to download hadoop from.",
```

Comment:
Nitpick: This should just be "URL to download Hadoop from." since even the default URL is specified here. Capitalize the "Hadoop" too, please. :)